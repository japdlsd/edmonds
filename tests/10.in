60 200
1 13 2
1 22 10
1 26 5
1 41 7
1 49 5
1 53 1
1 57 3
1 58 6
2 4 2
2 25 10
2 27 5
2 59 1
3 11 7
3 13 8
3 20 6
3 28 6
3 30 10
3 49 8
3 53 8
3 54 2
3 57 10
4 19 4
4 21 10
4 24 7
4 25 9
4 29 6
4 33 4
4 47 3
4 57 8
5 9 10
5 12 7
5 13 10
5 18 2
5 32 1
5 49 8
6 21 7
6 33 6
6 54 3
6 55 6
7 27 6
7 30 1
7 46 1
8 13 4
8 19 9
8 27 8
8 29 6
8 34 9
8 38 6
8 49 5
8 54 1
8 59 7
9 20 4
9 27 3
9 44 2
9 59 3
10 32 7
10 38 9
10 46 10
10 59 9
11 18 2
11 23 6
11 27 8
11 32 1
11 55 4
12 23 3
12 27 8
12 28 2
12 41 2
12 50 2
12 51 6
12 57 8
13 21 3
13 32 7
13 43 6
14 30 8
14 45 6
14 56 10
15 24 10
15 29 10
15 56 2
15 58 8
16 25 3
16 27 8
16 28 1
16 29 8
16 31 3
16 45 3
16 46 6
16 59 2
17 20 10
17 33 3
18 20 7
18 27 9
18 46 8
19 40 7
19 50 10
19 57 9
20 21 8
20 26 6
20 31 7
21 22 10
21 24 6
22 24 4
22 28 4
22 31 6
22 42 1
22 55 9
23 24 8
23 43 3
23 44 3
24 31 9
24 36 1
24 57 7
25 40 3
25 41 7
25 50 8
25 60 5
26 33 4
26 34 7
26 35 5
26 43 7
26 45 9
26 56 9
27 40 5
27 48 10
27 55 10
28 45 2
28 54 5
28 60 5
29 34 2
29 41 3
29 45 1
29 46 3
29 48 2
29 54 3
30 37 4
30 40 8
30 43 2
30 44 3
30 50 3
30 53 1
31 37 5
31 38 10
31 39 7
31 41 1
31 46 7
31 52 5
32 35 9
32 39 7
32 51 9
32 55 5
32 57 3
32 58 7
33 36 6
33 45 9
34 38 2
35 41 1
35 47 4
35 51 8
35 55 3
35 58 7
36 46 10
36 52 9
37 41 7
37 42 6
37 46 4
37 47 4
37 55 5
38 43 9
39 52 9
39 58 3
40 45 2
40 48 1
40 56 2
41 44 3
41 46 1
41 52 3
41 53 1
41 60 5
42 43 7
42 44 1
42 60 7
43 45 5
43 60 4
44 51 10
45 56 6
46 48 3
46 56 5
46 57 4
46 58 4
47 53 1
47 60 2
48 50 2
50 59 7
51 53 8
51 57 2
54 57 10
56 57 7
57 59 10
58 59 5
