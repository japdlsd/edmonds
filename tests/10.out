88
51 57
40 48
39 58
37 47
35 41
31 52
30 43
29 34
25 60
24 36
23 44
22 42
20 26
17 33
16 28
15 56
14 45
13 21
12 50
11 32
10 38
9 27
8 49
7 46
6 55
5 18
4 19
3 54
2 59
1 53

